network/network.go

```diff
@@ -341,26 +340,22 @@ func (n *Network) Dial(address string) (net.Conn, error) {
 
 // Accept handles peer registration and processes incoming message streams.
 func (n *Network) Accept(incoming net.Conn) {
-	var outgoing net.Conn
-
 	var client *PeerClient
 	var clientInit sync.Once
 
 	recvWindow := NewRecvWindow(n.opts.recvWindowSize)
 
 	// Cleanup connections when we are done with them.
 	defer func() {
+		time.Sleep(1 * time.Second)
```

Comment:
Why the sleep for 1 second?

Reply:
I've found a race cond during peer disconnect. Adding this timeout solves a problem.
For sure I'm going to fix it for real, now it's a hack 😉